h5py/h5p.pyx

```diff
@@ -1130,6 +1130,37 @@ cdef class PropFAID(PropInstanceID):
         """
         H5Pset_fapl_sec2(self.id)
 
+    if DIRECT_VFD:
+        @with_phil
+        def set_fapl_direct(self, size_t alignment=0, size_t block_size=0, size_t cbuf_size=0):
+            """(size_talignment, size_t block_size, size_t cbuf_size)
+
+            Select the "direct" driver (h5fd.DIRECT).
+
+            Parameters:
+                hid_t fapl_id       IN: File access property list identifier
+                size_t alignment    IN: Required memory alignment boundary
+                size_t block_size   IN: File system block size
+                size_t cbuf_size    IN: Copy buffer size
+
+            Properites with value of 0 indicate that the HDF5 library should
+            choose the value.
```

Comment:
Here is the reference in HDF5's source that will populate non-zero values
https://github.com/HDFGroup/hdf5/blob/bf27121b5d01939c819ac8b5b96101a16175c4de/src/H5FDdirect.c#L352